recipes/shinywidgets/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "shinywidgets" %}
+{% set version = "0.3.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/shinywidgets-{{ version }}.tar.gz
+  sha256: c5c767f6d4176f90bbcf756657d307d47b96e6efdd9535a33c8b0d7821068a89
+
+build:
+  skip: true  # [py<38]
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python
+    - setuptools
+    - pip
+  run:
+    - python
+    - ipywidgets >=7.6.5
+    - jupyter_core
+    - shiny >=0.6.1.9005
+    - python-dateutil >=2.8.2
+    - importlib-metadata >=4.8.3,<5  # [py<38]
```

Comment:
```suggestion
```

Reply:
This package metadata is wrong, it says it is py>=38 but requires a package from py<38. If you can, please open an issue upstream.

Reply:
Ah good catch. I'll submit an issue upstream.